engines/griffon/cutscenes.cpp

```diff
@@ -190,7 +190,7 @@ void GriffonEngine::intro() {
 	Common::TextToSpeechManager *_ttsMan = g_system->getTextToSpeechManager();
 	if (_ttsMan != nullptr) {
 		Common::String result = story[0];
-		for(int i=10; i< ARRAYSIZE(story); ++i){
+		for(int i = 10; i < ARRAYSIZE(story); ++i){
```

Comment:
This is not correct. Please pay attention to the details. As it was pointed out, the correct formatting is
`for (int i = 10; i < ARRAYSIZE(story); ++i) {`